Wrappers/Python/cil/optimisation/operators/BlurringOperator.py

```diff
@@ -41,9 +41,9 @@ def __init__(self, PSF, geometry):
         else:
             raise TypeError('PSF must be a number array with same number of dimensions as geometry.')
 
-        if not (isinstance(geometry,cil.framework.framework.ImageGeometry) or \
-                isinstance(geometry,cil.framework.framework.AcquisitionGeometry)):
-            raise TypeError('geometry must be an ImageGeometry or AcquisitionGeometry.')
+        #if not (isinstance(geometry,cil.framework.framework.ImageGeometry) or \
+        #        isinstance(geometry,cil.framework.framework.AcquisitionGeometry)):
+        #    raise TypeError('geometry must be an ImageGeometry or AcquisitionGeometry.')
```

Comment:
can we delete instead of commenting or is it likely to be reinstated?

are we unable to add a check for the specific sirf datacontainer type because the user might not have sirf?

Reply:
presumably we don't want to actually insist on it being a SIRF type either... Best is probably:

```python
        valid_types = [ImageGeometry, AcquisitionGeometry]
        try:
            import SIRFDataContainer
        except ImportError:
            pass
        else:
            valid_types += [SIRFDataContainer]
        if not isinstance(geometry, tuple(valid_types)):
            warnings.warn('unknown geometry type')
```

Reply:
@samdporter - I updated based on Casper's suggestions. I can't test on SIRF easily - does this work for you?

Reply:
need to delete `geometry` from `expected = [geometry,cil.framework.framework.ImageGeometry, geometry,cil.framework.framework.AcquisitionGeometry]`, I think

Reply:
Sorry @samdporter - try again? Thanks

Reply:
Looks good